app/api/routes/resource_modification.py

```diff
@@ -99,7 +101,9 @@ def get_unique_resource_languages_as_strings():
         db.session.commit()
 
         return utils.standardize_response(
-            payload=dict(data=resource.serialize),
+            payload=dict(
+                data=resource.serialize(g.auth_key.apikey)
```

Comment:
Nit: isn't this in the same function as above, and so we can reference it as `apikey` rather than `g.auth_key.apikey`?